host-vm1.yml

```diff
@@ -11,20 +11,20 @@
     name: buildbot
     hostname: buildbot.buildbot.net
     ip_address:
-    - 'lo0|127.0.0.239'       # localhost, must be first
-    - 'lagg0|140.211.10.239'
-    - 'vlan0|192.168.80.239'
+    - 'lo0|{{ hosts.buildbot.lo0 }}'    # localhost, must be first
+    - 'lagg0|{{ hosts.buildbot.lagg0 }}'
+    - 'vlan0|{{ hosts.buildbot.vlan0 }}'
   - role: jail
     name: docs
     hostname: docs.buildbot.net
     ip_address:
-    - 'lagg0|140.211.10.237'
-    - 'vlan0|192.168.80.237'
+    - 'lagg0|{{ hosts.docs.lagg0 }}'
+    - 'vlan0|{{ hosts.docs.vlan0 }}'
   - role: jail
     name: www
     hostname: www.buildbot.net
     ip_address:
-    - 'lagg0|140.211.10.238'
-    - 'vlan0|192.168.80.238'
+    - 'lagg0|{{ hosts.www.lagg0 }}'
+    - 'vlan0|{{ hosts.www.vlan0 }}'
```

Comment:
Would be nice to be able to autogenerate this. Perhaps the role should be changed to take a mapping of interface to IP? Then I think we could do `ip_address: {{ hosts.www }}`.